exercises/practice/bob/.approaches/performance/snippet.txt

```diff
@@ -0,0 +1,16 @@
+    // uses System.Linq;
```

Comment:
I agree. For performance I would be more interested in the performance table than the code that generates it. Maybe even collapse the code that creates it under a `details` section that you can expand if you want to play with it yourself.

Reply:
That would mean changing the snippet to markdown, as we currently automatically add syntax highlighting for code. This might still be a good time to do that, as we have so few approaches.

@iHiD ?

Reply:
I think making it an article instead of an approach makes sense, as long as it is under a category such that not every Dig Deeper is required to have a Performance article.

Reply:
I like article. Let me work on that.